src/bp-moderation/bp-moderation-admin.php

```diff
@@ -61,5 +70,201 @@ function bp_moderation_admin_menu_order( $custom_menus = array() ) {
  * @since BuddyBoss 1.5.4
  */
 function bp_moderation_admin_load() {
+	global $bp_moderation_list_table;
+
+	$doaction = bp_admin_list_table_current_bulk_action();
+
+	/**
+	 * Fires at top of Moderation admin page.
+	 *
+	 * @since BuddyBoss 1.5.4
+	 *
+	 * @param string $doaction Current $_GET action being performed in admin screen.
+	 */
+	do_action( 'bp_moderation_admin_load', $doaction );
+
+	// Create the Moderation screen list table.
+	$bp_moderation_list_table = new BP_Moderation_List_Table();
+
+	// The per_page screen option.
+	add_screen_option( 'per_page', array( 'label' => __( 'Moderation Request', 'buddyboss' ) ) );
+
+	// Help panel - overview text.
+	get_current_screen()->add_help_tab(
+		array(
+			'id'      => 'bp-moderation-overview',
+			'title'   => __( 'Overview', 'buddyboss' ),
+			'content' =>
+				'<p>' . __( 'Moderation overview line 1', 'buddyboss' ) . '</p>' .
+				'<p>' . __( 'Moderation overview line 2', 'buddyboss' ) . '</p>',
+		)
+	);
+
+	// Help panel - sidebar links.
+	get_current_screen()->set_help_sidebar(
+		'<p><strong>' . __( 'For more information:', 'buddyboss' ) . '</strong></p>' .
+		'<p>' . __( '<a href="https://www.buddyboss.com/resources/">Documentation</a>', 'buddyboss' ) . '</p>'
+	);
+
+	// Add accessible hidden heading and text for Activity screen pagination.
+	get_current_screen()->set_screen_reader_content(
+		array(
+			/* translators: accessibility text */
+			'heading_pagination' => __( 'Moderation list navigation', 'buddyboss' ),
+		)
+	);
+
+}
+
+/**
+ * Select the appropriate Moderation admin screen, and output it.
+ *
+ * @since BuddyBoss 1.5.4
+ */
+function bp_moderation_admin() {
+	// Added navigation tab on top.
+	if ( bp_core_get_groups_admin_tabs() ) { ?>
```

Comment:
@janakkaneriya007 I think `bp_core_get_groups_admin_tabs` is wrong funcation for moderation